### ISSUE TITLE
Part Mute für Split Sounds: Shift + Encoder/Dec/Inc im Split Sound Screen, Mute-"x" im Split-Symbol, Mute-Schalter im WebUI-Split-Sound-Tab

### ISSUE THREAD

**Issue by nonlinear-labs** [nonlinear-labs-dev/C15#2972]
Bei überlappenden Splits kann es wünschenswert sein, einen der Parts zu muten.

Mute-Zeichen sollten auch in den Part-Volume-Parameter-Screens sichtbar werden.
Idee: wenn der Part gemutet ist, sollte der Shift-Button die Parameter-Anzeige auf "Mute" umschalten und der Encoder kann zum UnMute verwendet werden.
Alternative: über den Edit-Button gehen...

**Comment by MatthiasSeeber** [comment]
Plus im Sound Screen (idealerweise sowohl im Split als auch Layer Sound): Part Muting mit Shift + Ecnoder

**Comment by davidm2894** [comment]
> Plus im Sound Screen (idealerweise sowohl im Split als auch Layer Sound): Part Muting mit Shift + Ecnoder

Evtl. stehe ich gerade auf dem Schlauch, aber das funktioniert doch schon über Dec/Inc bzw. Encoder?

**Comment by nonlinear-labs** [comment]
Ja, es geht auch eigentlich nur um das Muten eines Split-Parts. Das gibt es bisher noch gar nicht. Dies ist allerdings wohl nur interessant, wenn sich Splitbereiche überlappen. Für diesen Fall könnte Shift+Inc/Dec nützlich sein.
Ansonsten ist es vielleicht noch interessant, im Part-Volume auch eine Mute/UnMute-Aktion anzubieten. Weil das ja die Stelle (und der Parameter) ist wo die Mute-Wirkung überlagert wird.

**Comment by MatthiasSeeber** [comment]
Ich halte es für sinnvoll, das Muting Feature im Sound Screen bei beiden Dual-Sound-Typen auf gleiche Art und Weise zu implementieren. Da im Split Sound Inc/Dec und Encoder bereits den Split Point steuern, kommt dort also zweifelsohne nur Shift infrage. Im Layer Sound gibt es derzeit keine Inc/Dec/Encoder Standard-Belegung, aber es wäre doch komisch wenn ich im Split Shift drücken muss, im Layer Sound aber nicht.

**Comment by nonlinear-labs** [comment]
Der exotische Fall, dass jemand eine Split-Part muten will, rechtfertigt es für mich nicht, das häufig benutzte Muting von Layer-Parts unbequemer zu machen. Wenn da zusätzlich auch Shift + Inc/Dec wirkt: kein Problem.

**Comment by davidm2894** [comment]
Wenn man darüber geht, dass der Shift-Button den Parameter-Screen zu Mute/Unmute umschaltet, kommt man auch wieder in Konflikt mit dem Recall-Mechanismus, sofern Volume verstellt wurde. Ich weiß nicht, ob hier ein Edit-Mechanismus Sinn machen würde, da ist man schneller, wenn man den Part einfach über Shift+Dec auf -inf stellt. 

Ich finde auch, dadurch dass das Muten im Split-Mode so ein Corner-Case ist, sollte man nicht eine Mechanik verändern, die die User schon gewohnt sind.
